qml/TabLayout.qml

```diff
@@ -143,6 +149,7 @@ Item {
     // Add new tab button
     Rectangle {
         id: add_new_tab_button
+        z: 99
```

Comment:
`z` attribute from `tab_list`, `add_new_tab_button` and `remain_width_rect` not needed. Their areas never overlap (anchors properly fixed).